doc/rho.1

```diff
@@ -0,0 +1,408 @@
+
+.PP
+.B
+Listing and Showing Profiles
+.PP
+The 'list' commands lists the details for all configured profiles. The output includes the IP ranges, auth credentials, and ports for the profile.
+.PP
+.B rho profile list
+.PP
+The 'rho profile show' command is the same as the 'rho profile list' command, except that it returns details for a single specific profile. This is a handy command to verify edits to a profile.
+.PP
+.B rho profile show --name=
+.I profile
+
+.PP
+.B
+Deleting Profiles
+.PP
+Any or all profiles can be deleted using the 'clear' subcommand.
+.PP
+.B rho profile clear --name=
+.I name
+.B | --all
+
+.SS SCANNING
+The 'scan' command is the one that actually runs discovery on the network. This command scans all of the servers within the range, and then prints the information to a CSV file.
+.PP
+There are two ways to run a scan. The simplest way is to specify the profile to use:
+.PP
+.B rho scan --profile=
+.I profile_name
+.B --reportfile=
+.I file
+.B [--facts
+.I file or list of facts
+.B ]
+.PP
+.TP
+--profile=profile_name
+Gives the name of the profile to use to run the scan.
+.PP
+.TP
+--reportfile=file
+Prints the output to a comma-separated values (CSV) file instead of stdout.
+.PP
+.TP
+--facts fact1 fact2
+The list of facts that are returned in the scan output. The list below is included as an example and is not exhaustive. Please use the fact list command to get the full list of available facts.
+.nf
+cpu.count:number of processors
+cpu.cpu_family:cpu family
+cpu.model_name:cpu model name
+cpu.vendor_id:cpu vendor name
+dmi.bios-vendor:bios vendor name
+error:any errors that are found
+etc-release.etc-release:contents of /etc/release (or equivalent)
+instnum.instnum:installation number
+ip:ip address
+port:ssh port
+redhat-release.name:name of package that provides 'redhat-release'
+redhat-release.release:release of package that provides 'redhat-release'
+redhat-release.version:version of package that provides 'redhat-release'
+systemid.system_id:Red Hat Network system id
+systemid.username:Red Hat Network username
+virt.virt:host, guest, or baremetal
+virt.type:type of virtual system
+uname.all:uname -a (all)
+uname.hardware_platform:uname -i (hardware_platform)
+uname.hostname:uname -n (hostname)
+uname.kernel:uname -r (kernel)
+uname.os:uname -s (os)
+uname.processor:uname -p (processor)
+.fi
+.PP
+
+.SS OPTIONS FOR ALL COMMANDS
+.PP
+A the following option is allowed with every command for rho.
+.PP
+.TP
+--help
+This prints the help for the rho command or subcommand.
+
+
+.SH OUTPUT
+When the scan runs successfully, it outputs a CSV file that can be used to inventory the servers on the network. The output file has the following format:
+.nf
+ip,port,OS,kernel,processor,platform,release name,release version,release number,system ID,username,instnum,release,CPU count,CPU vendor,CPU model,BIOS vendor,virtual guest/host,virtual type,auth type,auth username,auth name,error
```

Comment:
Let's not include the CSV header. It will change over time, and I don't think it adds much here.